### DISCUSSION TITLE
file  "onlyMMirroir.exe" contains a Gen:Variant.Lazy.328671

### DISCUSSION THREAD

**Discussion by nateleon** [AntonyCorbett/OnlyM#477] [Q&A]
the "onlyMMirroir.exe" file contains a Gen:Variant.Lazy.328671 is this a false positive or something else ?

**Comment by AntonyCorbett** [comment]
@nateleon Please refer to my comments here:

https://github.com/AntonyCorbett/OnlyM/wiki/Malware
